js/src/popover.js

```diff
@@ -30,7 +30,7 @@ const Default = {
   content: '',
   template: '<div class="popover" role="tooltip">' +
               '<div class="popover-arrow"></div>' +
-                '<h3 class="popover-header"></h3>' +
+              '<h3 class="popover-header"></h3>' +
```

Comment:
@XhmikosR, I removed the space intentionally since h3 is not a nested element here :slightly_smiling_face:

Reply:
Oops, sorry, let me revert it